Xamarin.Forms.Platform.Tizen/Extensions/GeometryExtensions.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms.Shapes;
+using SkiaSharp;
+using FormsRectangle = Xamarin.Forms.Rectangle;
+
+namespace Xamarin.Forms.Platform.Tizen
+{
+    public static class GeometryExtensions
+    {
+        public static SKPath ToSKPath(this Geometry geometry)
+        {
+            SKPath path = new SKPath();
+
+            if (geometry is LineGeometry)
```

Comment:
The method seems to be too long. How about extracting methods using polymorphism for readability?
```csharp
private static SKPath MakeGeometry(LineGeometry geometry) {}
private static SKPath MakeGeometry(RectangleGeometry geometry) {}
private static SKPath MakeGeometry(EllipseGeometry) {} 
....
...
```

Reply:
I agreed on extracting methods. I will update.